config/initializers/active_storage.rb

```diff
@@ -0,0 +1,4 @@
+Rails.application.config.active_storage.content_types_allowed_inline =
```

Comment:
```suggestion
Rails.application.config.active_storage.content_types_allowed_inline +=
```
Let's not overwrite existing allowed content here

Reply:
Good spot, thanks, corrected.